.mergify/config.yml

```diff
@@ -6,7 +6,7 @@ pull_request_rules:
       label:
         add: [contribution/core]
     conditions:
-      - author~=^(eladb|RomainMuller|garnaat|nija-at|shivlaks|skinny85|rix0rrr|NGL321|Jerry-AWS|SomayaB|MrArnoldPalmer|NetaNir|iliapolo|njlynch)$
+      - author~=^(eladb|RomainMuller|garnaat|nija-at|shivlaks|skinny85|rix0rrr|NGL321|Jerry-AWS|SomayaB|MrArnoldPalmer|NetaNir|iliapolo|njlynch|madeline-k)$
```

Comment:
Can you kick out `shivlaks` while you're doing changes here, since he's no longer on the CDK team?

Reply:
done